internals/testing/test-bundler.js

```diff
@@ -6,8 +6,3 @@ import 'babel-polyfill';
 import chai from 'chai';
 import chaiEnzyme from 'chai-enzyme';
 chai.use(chaiEnzyme());
-
-// Include all .js files under `app`, except app.js, reducers.js, and routes.js.
-// This is for isparta code coverage
-const context = require.context('../../app', true, /^^((?!(app|reducers|routes)).)*\.js$/);
```

Comment:
oh yeah. i forgot about coverage.
this line is no longer needed, because we don't need to require/bundle files in order to generate coverage for them. I'll update the PR